### DISCUSSION TITLE
皮卡鱼和其他软件输棋记录

### DISCUSSION THREAD

**Discussion by IcePrince710** [official-pikafish/Pikafish#77] [General]
皮卡鱼输棋不多，但是还是存在bug,出现输棋。大家对皮卡鱼输棋的记录是怎么分析的？这问题出在哪儿

**Comment by skystarspython** [comment]
每个引擎都不是完美的，偶尔有输赢都是正常的，这不能算是bug

**Reply by skystarspython** [reply]
照你这样说，所有的象棋引擎都有bug

**Comment by anarki999** [comment]
試試設 MultiPV 2，但建議你是強机級別，i9 12代以後已經可以~
